docs/changehistory/NextVersion.md

```diff
@@ -119,11 +121,29 @@ All three `nativeDb` fields and `IModelHost.platform` have always been `@interna
 | `ElectronApp.callDialog`            | [ElectronApp.dialogIpc]($electron)                        |
 | `ElectronHost.getWindowSizeSetting` | [ElectronHost.getWindowSizeAndPositionSetting]($electron) |
 
+### API removals
+
+The following APIs have been removed:
```

Comment:
I feel like this is misleading (in a way that could easily alarm users). These APIs weren't exactly removed; it's just that they are no longer re-exported. Even if someone reads this whole section, I'm not sure it's obvious what was really done.

Reply:
Yea i wasnt sure the best way to categorize this. I will mention we no longer re-export these apis.